config/config.yaml

```diff
@@ -18,6 +18,7 @@ kraken:
             - scenarios/openshift/regex_openshift_pod_kill.yml
             - scenarios/openshift/vmware_node_scenarios.yml
             - scenarios/openshift/network_chaos_ingress.yml
+            - scenarios/openshift/pod_outage.yml
```

Comment:
s/pod_outage/pod_network_outage to make sure we don't confuse it with pod scenarios. Thoughts?

Reply:
Agreed, changed to pod_network_outage in the latest commit.